src/audiovideocontroller/AudioVideoController.ts

```diff
@@ -117,5 +117,5 @@ export default interface AudioVideoController extends AudioVideoControllerFacade
   /**
    * Returns the [[EventController]] for this audio-video controller.
    */
-  readonly eventController?: EventController;
+  readonly eventController: EventController;
```

Comment:
Since, eventController is optional through constructor for DAV, should this still be optional even though MeetingSession will always send one?

Reply:
Yes this should be.